README.md

```diff
@@ -88,16 +88,15 @@ This distribution currently supports:
 
 ## Supported Kubernetes distributions
 
-This helm chart is tested and works with default configurations on the following
-Kubernetes distributions:
+The Helm chart works with default configurations of the main Kubernetes distributions. Use actively supported versions:
 
-- [Vanilla (unmodified version) Kubernetes](https://kubernetes.io)
-- [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks)
+- [Vanilla (unmodified version) Kubernetes](https://endoflife.date/kubernetes)
```

Comment:
endoflife.date is a great service, but I'd think linking to primary sources is preferable.

Reply:
I recommend keeping the "End of Life" section as it provides valuable links to the primary source at the beginning of each page. Using "End of Life" also standardizes the format for presenting version support across various distributions, making it easier to understand. Parsing the version support documentation for each distribution at the source can be overwhelming. Open to other input.

Reply:
Done